tests/unit/lms/lti_test.py

```diff
@@ -0,0 +1,129 @@
+import pytest
+from pytest import param
+
+from lms.lti import LTI13Params, _ParamMapping
+
+
+def sample_function(value):
+    return value * 2
+
+
+class TestLTI13Params:
+    @pytest.mark.parametrize(
```

Comment:
Testing the basic mechanism of the class here, independently of any LTI mapping.